packages/core/src/layout/Header/Header.tsx

```diff
@@ -104,16 +104,11 @@ const TypeFragment: FC<TypeFragmentProps> = ({ type, typeLink, classes }) => {
   }
 
   if (!typeLink) {
-    return (
-      // </Link>
-      <Typography className={classes.type}>{type}</Typography>
-    );
+    // TODO: Add breadcrumps.
```

Comment:
breadcrumbs*

Reply:
Some serious nitpicking going on here 😬

Reply:
You say that now, but when we get a PR adding breadcrumps you won't be laughing!

Reply:
Now I'm laughing at least! 😄